### PR TITLE
Ros update location

### PR DESCRIPTION
**Pull request by cburnout**
# Description

This adds a rover dockerfile that lets us use our endpoints from our mocks so we can see how our system works. It uses env variable to know where to send data and who the user is this is so we can set env variables on the pis and not have to compile things into the system

Fixes # (issue)
ill make one
## Bug Fix or Feature

- [ ] Bug
- [ ] Feature
- [ ] Other

# Testing Instructions

- [ ] Do A
- [ ] Then B
- [ ] See if C breaks

## Linting

- [ ] I've linted my code

### CODE REVIEW

File: ARSIS-Rover/rover.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+TELEMETRY_API=172.17.0.1:8080
```
Comment (Aetheris743) [inline ARSIS-Rover/rover.env:1]: How should we handle this for deployment/testing?

File: ARSIS-Unity/UserSettings/Layouts/default-2021.dwlt
Comment (Aetheris743) [inline ARSIS-Unity/UserSettings/Layouts/default-2021.dwlt:1]: Did you mean to edit the default unity window layout?

File: ARSIS-Rover/ros/catkin_ws/src/vision_kit/scripts/listener.py
Comment (Aetheris743) [inline ARSIS-Rover/ros/catkin_ws/src/vision_kit/scripts/listener.py:1]: I don't think I understand why this file is here. How does it fit into the vision kit sending data?

File: ARSIS-Rover/ros/catkin_ws/src/vision_kit/scripts/location_updater.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+def main():
+    location_updater = LocationUpdater()
+
+    rospy.init_node("location_updater", anonymous=True)
+
+    rospy.Subscriber("heading", Float32, location_updater.heading_callback)
+    rospy.Subscriber("lla", LLA, location_updater.lla_callback)
+
+    # spin() simply keeps python from exiting until this node is stopped
```
Comment (Aetheris743) [inline ARSIS-Rover/ros/catkin_ws/src/vision_kit/scripts/location_updater.py:52]: Is this leftover from an earlier version?

File: ARSIS-Rover/rover.env (hunk 1 above)
Comment (Aetheris743) [reply ARSIS-Rover/rover.env:1]: We could probably use local DNS using the access point DHCP configuration

File: ARSIS-Rover/ros/catkin_ws/src/vision_kit/scripts/location_updater.py (hunk 2 above)
Comment (Aetheris743) [reply ARSIS-Rover/ros/catkin_ws/src/vision_kit/scripts/location_updater.py:52]: Or did you mean to add `spin()` somewhere else

**Review by Aetheris743 — APPROVED** [review]
Looks good to me.

File: ARSIS-Rover/rover.env (hunk 1 above)
Comment (cburnell) [reply ARSIS-Rover/rover.env:1]: This is so when we deploy it to a pi we have an easy way of setting where to look for the api

File: ARSIS-Unity/UserSettings/Layouts/default-2021.dwlt
Comment (cburnell) [reply ARSIS-Unity/UserSettings/Layouts/default-2021.dwlt:1]: I did not

File: ARSIS-Rover/ros/catkin_ws/src/vision_kit/scripts/listener.py
Comment (cburnell) [reply ARSIS-Rover/ros/catkin_ws/src/vision_kit/scripts/listener.py:1]: This was cruft from just a listener talker example and included so people can see the basics for pub/sub

File: ARSIS-Rover/ros/catkin_ws/src/vision_kit/scripts/location_updater.py (hunk 2 above)
Comment (cburnell) [reply ARSIS-Rover/ros/catkin_ws/src/vision_kit/scripts/location_updater.py:52]: Leftover from an earlier version. copy paste mixup from the previous examples

**Pull request merged**